src/main/java/org/datadog/jenkins/plugins/datadog/tracer/PythonConfigurator.java

```diff
@@ -0,0 +1,93 @@
+package org.datadog.jenkins.plugins.datadog.tracer;
+
+import hudson.FilePath;
+import hudson.model.Node;
+import hudson.remoting.VirtualChannel;
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import jenkins.MasterToSlaveFileCallable;
+import org.datadog.jenkins.plugins.datadog.util.ShellCommandExecutor;
+
+final class PythonConfigurator implements TracerConfigurator {
+
+    private static final Logger LOGGER = Logger.getLogger(PythonConfigurator.class.getName());
+
+    @Override
+    public Map<String, String> configure(DatadogTracerJobProperty<?> tracerConfig, Node node, FilePath workspacePath, Map<String, String> envs) throws Exception {
+        try {
+            String pipVersion = workspacePath.act(new GetPipVersion());
+            LOGGER.log(Level.FINE, "Got pip version " + pipVersion + " from " + workspacePath + " on " + node);
+
+            String installTracerOutput = workspacePath.act(new InstallTracer());
+            LOGGER.log(Level.FINE, "Trace installed in " + workspacePath + " on " + node + "; output: " + installTracerOutput);
+
+            String tracerLocation = getTracerLocation(workspacePath);
+
+            Map<String, String> variables = new HashMap<>();
+            variables.put("PYTEST_ADDOPTS", PropertyUtils.prepend(envs, "PYTEST_ADDOPTS", "--ddtrace"));
+            variables.put("PYTHONPATH", PropertyUtils.prepend(envs, "PYTHONPATH", tracerLocation + ":"));
+            return variables;
+
+        } catch (IOException e) {
+            LOGGER.log(Level.INFO, "Exception while trying to get node version from " + workspacePath + " on " + node + "; assuming node is not installed", e);
```

Comment:
```suggestion
            LOGGER.log(Level.INFO, "Exception while trying to get python version from " + workspacePath + " on " + node + "; assuming python is not installed", e);
```
I'm also not sure if this should be a catch-all: both  `GetPipVersion` and `InstallTracer` can throw an `IOException`, they can also throw an `InterruptedException`

I also think this should be a higher severity, maybe `warn`?

Reply:
Fixed the typo